src/styles/utils/variable.scss

```diff
@@ -0,0 +1,16 @@
+/* colors */
+$colorPrimary:#00acdc;
+$colorAccent: #060b35;
+$colorText: #616070;
+
+/* timing */
+$transitionDuration: 0.5s;
+
+/*catalog grid */
+$gapX: 50px;
+$gapY: 48px;
```

Comment:
you'll need to adjust both gaps by -2px - you'll see that on test results
mock indeed has 48 and 50, yet they're impacted by 1px borders (2 borders on X and Y axis)

Reply:
@meforoma  changes implemented however test has not been passed